### PR TITLE
Smoke: Separated scripts for GetVD and UpdateTurnList from smoke API script

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
Separated scripts for GetVehicleData and UpdateTurnList from smoke API script

### CODE REVIEW

File: test_scripts/Smoke/API/026_GetVehicleData_PositiveCase.lua [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    powerModeStatus = "KEY_OUT"
+  },
+  myKey = {
+    e911Override = "NO_DATA_EXISTS"
+  }
+}
+
+local paramsList = {}
+local requestParams = { }
+for k, _ in pairs(vehicleDataValues) do
```
Comment (AKalinich-Luxoft) [inline test_scripts/Smoke/API/026_GetVehicleData_PositiveCase.lua]: @GetmanetsIrina propose to move this loop to `setVDRequest` local function and call it from `getVD`

File: test_scripts/Smoke/API/026_GetVehicleData_PositiveCase.lua [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  responseUiParams = vehicleDataValues,
+}
+
+--[[ Local Functions ]]
+local function ptuUpdateFunc(tbl)
+  local SVDgroup = {
+    rpcs = {
+      GetVehicleData = {
+        hmi_levels = { "BACKGROUND", "FULL", "LIMITED" },
+        --For now commented because SDL does not process successfully list of parameters with more then 24 items
```
Comment (AKalinich-Luxoft) [inline test_scripts/Smoke/API/026_GetVehicleData_PositiveCase.lua]: @GetmanetsIrina you could just omit list of parameters - then all parameters for this RPC will be allowed

File: test_scripts/Smoke/API/027_UpdateTurnList_PositiveCase.lua [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+responseUiParams.turnList[1].turnIcon.value = commonSmoke.getPathToFileInStorage(requestParams.turnList[1].turnIcon.value)
+responseUiParams.softButtons[1].image.value = commonSmoke.getPathToFileInStorage(requestParams.softButtons[1].image.value)
+
+local allParams = {
+  requestParams = requestParams,
+  responseUiParams = responseUiParams,
+}
+
+--[[ Local Functions ]]
+local function ptuUpdateFunc(tbl)
```
Comment (AKalinich-Luxoft) [inline test_scripts/Smoke/API/027_UpdateTurnList_PositiveCase.lua]: @GetmanetsIrina should be renamed to `ptUpdateFunc` as 'u' already means 'update'

File: test_scripts/Smoke/API/026_GetVehicleData_PositiveCase.lua (hunk 1 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/026_GetVehicleData_PositiveCase.lua]: @AKalinich-Luxoft , please find update in 9c0827d09fcbc036e9df4633cac640738392824d.

File: test_scripts/Smoke/API/026_GetVehicleData_PositiveCase.lua (hunk 2 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/026_GetVehicleData_PositiveCase.lua]: @AKalinich-Luxoft , Done in 9c0827d09fcbc036e9df4633cac640738392824d

File: test_scripts/Smoke/API/027_UpdateTurnList_PositiveCase.lua (hunk 3 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/027_UpdateTurnList_PositiveCase.lua]: @AKalinich-Luxoft , Done in 9c0827d09fcbc036e9df4633cac640738392824d.

**Pull request merged**